pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["flit_core >=2,<4"]
+requires = ["flit_core >=2,<4", "python-dotenv"]
```

Comment:
Should this be version pinned?

Reply:
I will remove this in the latest commit as I already have defined this dependency in setup.cfg